### DISCUSSION TITLE
when will multitouch and touch buttons for some modes be added to the game?

### DISCUSSION THREAD

**Discussion by Derilat** [ppy/osu#16042] [Q&A] [answered]
I am wildly annoyed by this question. People who do not have a physical keyboard suffer from the fact that they cannot normally or generally play some modes, or stream on OSU maps.

**Comment by bdach** [comment, marked as answer]
and i am mildly annoyed to answer this for the hundredth time: mobile is not a current priority. we know it needs doing, but it's not a priority right now, the priority is desktop first and foremost. see https://github.com/ppy/osu/issues/4201 and the *many many MANY* linked issues and discussions.
